src/style.css

```diff
@@ -0,0 +1,80 @@
+/* stylelint-disable color-hex-case */
```

Comment:
No need to disable the eslint. You can just fix the problem and convert color into the proper format